### ISSUE TITLE
Stopped working

### ISSUE THREAD

**Issue by DagBjorndal** [serayuzgur/crates#225]
**Describe the bug**
Stopped showing anything. 

**To Reproduce**
1. Insert name of any crate from crates.io into Cargo.toml under [dependencies]
2. Append the usual equal sign with spaces and quotation marks (eg. tokio = "")
3. Save the file
4. Hover over the quotation marks
5. Observe that no pop-up is shown

**Expected behavior**
A pop-up listing all version for the dependency should be produced.

**Desktop (please complete the following information):**
 - Raspberry Pi 5
 - OS: Raspberry Pi OS
 - Version: 6.1.0-rpi8-rpi-2712

**Comment by serayuzgur** [comment]
Please share error log from console.

**Comment by kmindi** [comment]
I noticed that the Cargo.toml was only recognized by VS Code as "Plain Text". When I installed an extension that adds support for TOML, TOML was detected for the Cargo.toml file and this crates plugin showed again the expected inline version icons.

**Comment by djc** [comment]
I've also been seeing some issues -- however, it seems to be inconsistent? In some VS Code workspaces it works, in others it doesn't. Current hypothesis: it only works in Cargo workspaces, not in single-crate Cargo manifests, maybe?

**Comment by milen-denev** [comment]
Hello, it's not been working for me as well, since 0.6.6.

**Comment by Kidsunbo** [comment]
VSCode recognizes `Cargo.toml` as `Plain Text` and `crates` doesn't work with `Plain Text`. Just install `Even Better Toml` to make VSCode knows what `toml` is, and everything works.

**Comment by djc** [comment]
> VSCode recognizes `Cargo.toml` as `Plain Text` and `crates` doesn't work with `Plain Text`. Just install `Even Better Toml` to make VSCode knows what `toml` is, and everything works.

I don't think this is accurate -- I have "Even Better Toml" installed but disabled all the time and it still works.

**Comment by Kidsunbo** [comment]
> > VSCode recognizes `Cargo.toml` as `Plain Text` and `crates` doesn't work with `Plain Text`. Just install `Even Better Toml` to make VSCode knows what `toml` is, and everything works.
> 
> I don't think this is accurate -- I have "Even Better Toml" installed but disabled all the time and it still works.

That's weird. I just tried disabling the `Even Better Toml` and then `crates` didn't work immediately.
